### DISCUSSION TITLE
Additional updates in group with already merged PR are blocked

### DISCUSSION THREAD

**Discussion by maxbrunet** [renovatebot/renovate#16771] [Request Help] [answered]
When there are additional updates and a merged PR with the same title exists, they are blocked.

Not sure if this is a bug or a misconfiguration in this particular case:

> ## Ignored or Blocked
> 
> These are blocked by an existing closed PR and will not be recreated unless you click a checkbox below.
> 
> * [ ]  [chore(deps): pin dependencies](https://github.com/parca-dev/parca-agent/pull/491) (`actions/download-artifact`, `docker.io/goreleaser/goreleaser-cross`, `docker.io/library/busybox`, `gcr.io/distroless/base-debian11`)
> 
> * [ ]  [build(deps): update kubernetes to v0.24.3](https://github.com/parca-dev/parca-agent/pull/589) (`k8s.io/api`, `k8s.io/client-go`)

Configuration: [parca-dev/parca-agent//renovate.json](https://github.com/parca-dev/parca-agent/blob/cbada34ca378fa3fc7b2acce84b0d43e7d9bdb51/renovate.json) ([github>parca-dev/.github](https://github.com/parca-dev/.github/blob/59766be0c7e6b50ba1e13ae5e499144383d53833/default.json) preset)

I feel like the PR for the `renovate/pin-dependencies` branch should always be recreated, and if some dependencies should not be pinned, the config should be tweaked.

For a group, I understand that the PR could have been reverted, and Renovate should not try to update again, I cannot think of a way around it without breaking the feature.

It seems to have been reported in #16141, but somehow Renovate unblocked itself (or maybe a teammate checked the box and they did not notice)

**Comment by rarkins** [comment, marked as answer]
I think that previously merged PRs should not block creation of a new PR, however automerge should be disabled in such cases in case the user "rolled back" the updates after the initial merge. I'm not sure we have a feature request for this yet. The current approach is intended to stop "infinite" updating if the user merges, finds a problem, and rolls back.

**Reply by maxbrunet** [reply]
:+1: created #16820

**Reply by TheKangaroo** [reply]
Today I ran into this with renovate version 34.107.0.
The PR that fixed this https://github.com/renovatebot/renovate/pull/16898 was reverted one day later https://github.com/renovatebot/renovate/pull/17699 . So this issue still persists, right?

**Reply by viceice** [reply]
yes, issue reopened
